tutorials/plot_1_bycycle_philosophy.py

```diff
@@ -100,32 +94,29 @@
 
 # Different hyperparameter choices - filter length and center frequency and bandwidth
 f_alphas = [(6, 14), (8, 12), (9, 13)]
-n_secondss = [.4, .75, 1.2]
+n_seconds = [.4, .75, 1.2]
 
 amps = []
 phases = []
+
 for f_alpha in f_alphas:
-    for n_seconds_filter in n_secondss:
-        amp = amp_by_time(sig, fs, f_alpha, n_seconds=n_seconds_filter)
-        phase = phase_by_time(sig, fs, f_alpha, n_seconds=n_seconds_filter)
+
+    for n_second_filter in n_seconds:
+
+        amp = amp_by_time(sig, fs, f_alpha, n_seconds=n_second_filter)
+        phase = phase_by_time(sig, fs, f_alpha, n_seconds=n_second_filter)
+
         amps.append(amp)
         phases.append(phase)
 
-plt.figure(figsize=(12,2))
-for amp in amps:
-    plt.plot(times[tidx], amp[tidx])
-plt.xlim(tlim)
-plt.tight_layout()
-plt.show()
 
-####################################################################################################
+color_cycle = plt.rcParams['axes.prop_cycle'].by_key()['color']
+
+fig, axes = plt.subplots(figsize=(15, 6), nrows=2)
+
+plot_time_series(times, amps, ax=axes[0], colors=color_cycle, xlim=(2, 6))
```

Comment:
`plot_time_series` accepts a `xlim` but not `tlim`. Maybe the ndsp funcs could accept both `xlim` or `tlim`, interchangeably. Or I can change `tlim` in bycycle to `xlim`. I can make the change either way you prefer.

Reply:
Ohh, good point! 
What I was thinking of when I wrote this comment is just to write `xlim=tlim` in these lines, since we have the `tlim` variable declared. But now that you bring it up, ByCycle does use a `tlim` label for what is basically a `xlim` thing right? 

My vote would be consolidate on using `xlim` for plotting. Where-ever possible, I think we want to be consistent with the matplotlib API. 

The quirk is ByCycle is that we also have the functions that update DF values based on time ranges. For those functions, `xlim` isn't a good name. Maybe for visualizations, call it `xlim`, and for DF functions, call it `time_range` or similar?

Reply:
I'll switch to xlim in the plotting funcs.

Bycylce actually doesn't use tlim at all outside of plotting, which I didn't realize until now. All other functions expect signal/times to be limited prior to function calls.

Reply:
Okay, perfect, that all sounds good then!

Reply:
I just double checked - `plots/utils/apply_tlims` uses tlims - but it seems to me that this is a general utility function that could be used outside plotting. 

What do you think of updating to something like:
`def restrict_df_time_range(df, fs, time_range):`
And putting it into a general utility function?

Related, I sorta think that function can operate on DF's without also updating signals. A separate utility, if needed, could do `restrict_sig_time_range`. (Names are quick guesses - feel free to update).

Reply:
Good idea! I think moving it to a general utility makes since now that I've noticed there is not a `xlim` arg in `compute_features`, etc. I also think two separate funcs, one for `df` and a second for `sig`/`times`  would be helpful for time limiting in `compute_features`.